lib/ansible/module_utils/ec2.py

```diff
@@ -150,10 +150,8 @@ def get_aws_connection_info(module, boto3=False):
         if validate_certs:
             boto_params['verify'] = validate_certs
 
-        if profile_name:
-            boto_params['profile_name'] = profile_name
-
-
+        return region, ec2_url, profile_name, boto_params
```

Comment:
This appears to change the specification of get_aws_connection_info to return 4 values only when using boto3. 

Wouldn't you need to update every single module that calls `get_aws_connection_info`?